osu.Game/Screens/Play/ScreenWithBeatmapBackground.cs

```diff
@@ -17,6 +17,8 @@ public abstract class ScreenWithBeatmapBackground : OsuScreen
 
         public override bool AllowBeatmapRulesetChange => false;
 
+        public override bool AllowUserSeek => false;
```

Comment:
This must be at a `Player` level. At the moment this disallows seeking in replays.